NSS-Day-Cohort-53/tabloid-full-stack-always-puffy-impalas
# Ne new

#### Changes Made
1. Added Reaction.cs in Models
2. Added Add Functionality in ReactionRepository/IReactionRepository.cs in Repositories
3.Added Post Functionality in ReactionController.cs in Controllers
Added addReaction function in reactionManager in modules in vsCode
Added form AddReaction.js component in components in vsCode
Added addReaction route and in the navbar in vsCode

​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout ne-new
    ```
4. Open a new Terminal tab (⌘T) and navigate to the server directory.
5. Test app functionality.
    > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
    > Example: Run Api, Run npm start, Login, Select add reaction in the navBar then fill out form and click submit. Form should not submit if it isn't completely filled out. Check DB to see if reaction was added to the reaction table
6. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
